test/user.js

```diff
@@ -1,11 +1,14 @@
 const chai = require('chai'),
-  dictum = require('dictum.js'),
+  assert = require('chai').assert,
```

Comment:
you are already using should. Chai has 3 "ways" of using it, with assert, should or expect. Let's try not to mix them up.